benchmark_runner/common/ocp_resources/create_odf.py

```diff
@@ -52,14 +54,9 @@ def create_odf(self):
                                                       verify_cmd=r"""oc get pv -o jsonpath="{range .items[*]}{.metadata.name}{'\n'}{end}" | grep local | wc -l""",
                                                       count_openshift_storage=True)
                 if '07_subscription.yaml' in resource:
-                    # wait till get the patch
-                    self.wait_for_ocp_resource_create(resource='odf',
-                                                      verify_cmd="oc get InstallPlan -n openshift-storage -ojsonpath={.items[0].metadata.name}",
-                                                      status="install-")
-                    self.apply_patch(namespace='openshift-storage', resource='odf')
-                elif '08_storage_cluster.yaml' in resource:
                     # Must be run after installing the storage cluster because CSVs sometimes fail
```

Comment:
Looks like this comment has effectively moved from 08_storage_cluster.yaml to 07_subscription.yaml.

Reply:
Yes, in the past we used installPlanApproval: Manual and I have changed it to installPlanApproval: Automatic, that cause that issue, remove this comment.